lib/safe_code/validator/function_validators/phoenix.ex

```diff
@@ -1,15 +1,37 @@
 defmodule SafeCode.Validator.FunctionValidators.Phoenix do
   @behaviour SafeCode.Validator.FunctionValidators.Behaviour
 
+  def safe_function?(:my_component), do: true
+  def safe_function?(:rem), do: true
+  def safe_function?(:&), do: true
+  def safe_function?(:/), do: true
+  def safe_function?(:{}), do: true
   def safe_function?(_), do: false
 
+  def safe_module_function?(DockYardWeb.EmployeeView, :random_avatar_uri), do: true
+  def safe_module_function?(DockYardWeb.EmployeeView, :display_name), do: true
+  def safe_module_function?(DockYardWeb.PostView, :illustration), do: true
+  def safe_module_function?(DockYardWeb.PostView, :post_path), do: true
+  def safe_module_function?(DockYardWeb.PostView, :author_avatar_url), do: true
+  def safe_module_function?(DockYard.ClientLeads, :change_client_lead_form), do: true
```

Comment:
I need to think of a way to remove this, it's trying to validate AST like this one:
```elixir
{:., [line: 95], [{:__aliases__, [line: 95], [:DockYardWeb, :EmployeeView]}, :display_name]}
```

Reply:
@alexandrexaviersm `safe_code` allows the application using it to specify its own safe functions. We are doing this in Beacon, but I need to expose a way for users using Beacon to tag safe functions. I am working on a PR for this now. You can remove these lines from this PR.